lib/ui/components/qrstatuswidget.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:flutter/material.dart';
+import 'package:qrapp/models/seed.dart';
+import 'package:qrapp/models/seedstatus.dart';
+import 'package:intl/intl.dart';
+
+class QRStatusWidget extends StatefulWidget {
+  SeedStatus status;
```

Comment:
**bug**: widgets in flutter are immutable and the **all** should have `final` params and a `const` constructor. You should fix all of your widgets to lool like this:

```suggestion
  final SeedStatus status;
  final SeedModel? seed;
  
  const QRStatusWidget({
    Key? key,
    required this.status,
    required this.seed,
  }) : super(key: key);
```